### ISSUE TITLE
Support for Standardized Mean Difference output?

### ISSUE THREAD

**Issue by joelkuiper** [gertvv/gemtc#40]
I assume (with your new job and all, congratz!) that you won't have a tremendous amount of time implementing this. But maybe you know some literature so I can give it a stab and send a PR when/if done?

**Comment by gertvv** [comment]
This would be great actually, its on our roadmap, at least I think you mean *input*? It should be pretty straightforward. My preference would be for it to be a new likelihood, rather than having it be done through pre-processing of the data. Possibly "StandardizedNormal" works as the likelihood name?

**Comment by joelkuiper** [comment]
Yes I meant input! changed the title too. Potentially I have some code for doing it by pre-processing the data, I guess doing it with a new likelihood would probably entail lifting that code and putting it in the the dedicated likelihood code, or would you rather have JAGS deal with it?

**Comment by gertvv** [comment]
I have started a branch for this, some work remains (e.g. starting values), but it seems to work.

https://github.com/gertvv/gemtc/tree/feature/smd

If you have a good smallish example dataset, that would help. The only example I have is huge and confidential, so not great for a validation / regression test.
